### PR TITLE
Remove unused NoCredentialsDefinedException; spell fixes; simplify

### PR DESCRIPTION
**Pull request by adamdickmeiss**
These are minor suggested changes, reported by IntelliJ IDEA.

### CODE REVIEW

File: src/main/java/org/folio/edge/core/Handler.java [hunk 1]
```diff
@@ -180,6 +180,6 @@ protected void internalServerError(RoutingContext ctx, String msg) {
 
   @FunctionalInterface
   public interface TwoParamVoidFunction<A, B> {
-    public void apply(A a, B b);
+    void apply(A a, B b);
   }
```
Comment (julianladisch) [inline src/main/java/org/folio/edge/core/Handler.java:184]: Can TwoParamVoidFunction been replaced by java.util.function.BiConsumer?

File: src/main/java/org/folio/edge/core/cache/Cache.java [hunk 2]
```diff
@@ -69,14 +69,12 @@ private void prune() {
     logger.info("Cache size before pruning: {}", storage.size());
 
     LinkedHashMap<String, CacheValue<T>> updated = new LinkedHashMap<>(capacity);
-    Iterator<String> keyIter = storage.keySet().iterator();
-    while (keyIter.hasNext()) {
-      String key = keyIter.next();
-      CacheValue<T> val = storage.get(key);
+    for (Map.Entry<String, CacheValue<T>> entry : storage.entrySet()) {
```
Comment (julianladisch) [inline src/main/java/org/folio/edge/core/cache/Cache.java:72]: storage.forEach((key, val) -> {

**Review by julianladisch — APPROVED** [review]
Looks good, thanks!

File: src/main/java/org/folio/edge/core/Handler.java (hunk 1 above)
Comment (craigmcnally) [reply src/main/java/org/folio/edge/core/Handler.java:184]: It'd be a breaking change, right?  So if we're going to do this let's make it part of v4

**Review by craigmcnally — APPROVED** [review]

File: src/main/java/org/folio/edge/core/Handler.java (hunk 1 above)
Comment (adamdickmeiss) [reply src/main/java/org/folio/edge/core/Handler.java:184]: Public is redundant here. Not a breaking change

File: src/main/java/org/folio/edge/core/cache/Cache.java (hunk 2 above)
Comment (adamdickmeiss) [reply src/main/java/org/folio/edge/core/cache/Cache.java:72]: nice. Changed

**Pull request merged**